app/services/llamaparse_service.py

```
@@ -0,0 +1,827 @@
+            return result
+
+        except Exception as e:
+            logger.error(f"LlamaParse failed: {e}", exc_info=True)
+            result["errors"].append(str(e))
+            return result
+
+    def _build_parse_options(
+        self,
+        mode: ParseMode,
+        parsing_instructions: Optional[str],
+        extract_tables: bool,
```

Ruff (0.14.14)

[warning] Unused method argument: `extract_tables`

(ARG002)